src/GitVersion.MsBuild.Task/GitVersionTaskBase.cs

```diff
@@ -0,0 +1,13 @@
+namespace GitVersion.MsBuild.Task
+{
+    using Microsoft.Build.Framework;
+    using Microsoft.Build.Utilities;
+
+    public abstract class GitVersionTaskBase : Task
```

Comment:
Perhaps you can add this file with the same contents as the old `GitVersionTaskBase` and then move in the new content in two separate commits so Git detects it as a move?